pkgs/by-name/tl/tlock/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "tlock";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "eklairs";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-O6erxzanSO5BjMnSSmn89w9SA+xyHhp0SSDkCk5hp8Y=";
+    fetchSubmodules = true;
+  };
+
+  vendorHash = "sha256-G402CigSvloF/SI9Wbcts/So1impMUH5kroxDD/KKew=";
+
+  doCheck = false;
```

```suggestion
  doCheck = false;
```
Do add a comment explaining why the checks won't pass.